sdk/communication/azure-communication-chat/azure/communication/chat/_models.py

```diff
@@ -152,18 +200,19 @@ class ChatMessageContent(object):
     :ivar initiator: Chat message content for messages of types participantAdded or
      participantRemoved.
     :type initiator: CommunicationIdentifier
+    :ivar attachments: Chat message content for messages of type text or html
+    :type attachments: List[~azure.communication.chat.models.ChatAttachment]
```

Comment:
Looks like the type formatting throughout this class docstring needs updating:
```suggestion
    :vartype attachments: List[~azure.communication.chat.models.ChatAttachment]
```